1.4.2: build fauls with gcc 11

```
[  4%] Building CXX object CMakeFiles/ebml.dir/src/EbmlString.cpp.o
/usr/bin/g++ -Debml_EXPORTS -I/home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/x86_64-redhat-linux-gnu -I/home/tkloczko/rpmbuild/BUILD/libebml-1.4.2 -O2 -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fdata-sections -ffunction-sections -flto=auto -flto-partition=none -O2 -g -DNDEBUG -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -std=gnu++11 -o CMakeFiles/ebml.dir/src/EbmlString.cpp.o -c /home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/src/EbmlString.cpp
/home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/src/EbmlString.cpp: In member function ‘virtual filepos_t libebml::EbmlString::ReadData(libebml::IOCallback&, libebml::ScopeMode)’:
/home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/src/EbmlString.cpp:147:41: error: ‘numeric_limits’ is not a member of ‘std’
  147 |     auto Buffer = (GetSize() + 1 < std::numeric_limits<std::size_t>::max()) ? new (std::nothrow) char[GetSize() + 1] : nullptr;
      |                                         ^~~~~~~~~~~~~~
/home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/src/EbmlString.cpp:147:67: error: expected primary-expression before ‘>’ token
  147 |     auto Buffer = (GetSize() + 1 < std::numeric_limits<std::size_t>::max()) ? new (std::nothrow) char[GetSize() + 1] : nullptr;
      |                                                                   ^
/home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/src/EbmlString.cpp:147:70: error: ‘::max’ has not been declared; did you mean ‘std::max’?
  147 |     auto Buffer = (GetSize() + 1 < std::numeric_limits<std::size_t>::max()) ? new (std::nothrow) char[GetSize() + 1] : nullptr;
      |                                                                      ^~~
      |                                                                      std::max
In file included from /usr/include/c++/11/algorithm:62,
                 from /home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/ebml/EbmlEndian.h:41,
                 from /home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/ebml/EbmlTypes.h:38,
                 from /home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/ebml/EbmlString.h:41,
                 from /home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/src/EbmlString.cpp:38:
/usr/include/c++/11/bits/stl_algo.h:3467:5: note: ‘std::max’ declared here
 3467 |     max(initializer_list<_Tp> __l, _Compare __comp)
      |     ^~~
/home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/src/EbmlString.cpp:157:17: error: type ‘<type error>’ argument given to ‘delete’, expected pointer
  157 |       delete [] Buffer;
      |                 ^~~~~~
```

OK so after cut off Changelog part looks like that patch is fixing tha issue :)
```
Wrote: /home/tkloczko/rpmbuild/SRPMS/libebml-1.4.2-2.fc35.src.rpm
Wrote: /home/tkloczko/rpmbuild/RPMS/libebml-devel-1.4.2-2.fc35.x86_64.rpm
Wrote: /home/tkloczko/rpmbuild/RPMS/libebml-1.4.2-2.fc35.x86_64.rpm
Wrote: /home/tkloczko/rpmbuild/RPMS/libebml-debugsource-1.4.2-2.fc35.x86_64.rpm
Wrote: /home/tkloczko/rpmbuild/RPMS/libebml-debuginfo-1.4.2-2.fc35.x86_64.rpm
```
BTW two things:
I have in my spec file part:
```
%check
echo "%{name} package has no test suite"
```
Looks like libebml has no test suite which could be used n build stage. Am I right?

And seconf thing is warning durin build:
```
/home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/src/EbmlElement.cpp: In member function 'filepos_t libebml::EbmlElement::OverwriteData(libebml::IOCallback&, bool)':
/home/tkloczko/rpmbuild/BUILD/libebml-1.4.2/src/EbmlElement.cpp:713:8: warning: unused variable 'DataSize' [-Wunused-variable]
  713 |   auto DataSize   = GetSize();
      |        ^~~~~~~~
```

Nevertheless .. thank you :)
Feel free to close that ticket.

--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,7 @@
+2021-02-27  Moritz Bunkus  <mo@bunkus.online>
+
+        * Fix compilation with g++ 11. Fixes #80.
+
 2021-02-21 Steve Lhomme <slhomme@matroska.org>
 
         * Remove Coremake project files

--- a/src/EbmlString.cpp
+++ b/src/EbmlString.cpp
@@ -34,6 +34,7 @@
   \author Steve Lhomme     <robux4 @ users.sf.net>
 */
 #include <cassert>
+#include <limits>
 
 #include "ebml/EbmlString.h"
 

--- a/src/EbmlUnicodeString.cpp
+++ b/src/EbmlUnicodeString.cpp
@@ -36,6 +36,7 @@
 */
 
 #include <cassert>
+#include <limits>
 
 #include "ebml/EbmlUnicodeString.h"
 

